framework/decode/screenshot_handler.cpp

```diff
@@ -366,7 +367,7 @@ void ScreenshotHandler::WriteImage(const std::string&                      filen
                 if (result == VK_SUCCESS)
                 {
                     void* data = nullptr;
-                    result     = allocator->MapResourceMemoryDirect(
+                    result     = allocator->GetDirectAllocator().MapResourceMemory(
```

Comment:
This line is returning `VK_ERROR_MEMORY_MAP_FAILED` in this PR but returns `VK_SUCCESS` in `dev` branch.  This is why at least vkcube_95_105 test case is failing in Jenkins CI.